sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/TransformWithStateExec.scala

```diff
@@ -416,6 +429,8 @@ case class TransformWithStateExec(
       StatefulOperatorCustomSumMetric("numMapStateVars", "Number of map state variables"),
       StatefulOperatorCustomSumMetric("numDeletedStateVars", "Number of deleted state variables"),
       // metrics around timers
+      StatefulOperatorCustomSumMetric("timerProcessingTimeMs",
+        "Number of milliseconds taken to process all timers"),
```

Comment:
Could we add some test to verify presence of metric/non-zero value (maybe add to an existing test) ?

Reply:
Yea sounds good - lets add this test